test-data/typecheck/test_serializers.yml

```diff
@@ -8,4 +8,30 @@
             class Meta:
                 pass
         class SerializerC(SerializerA, SerializerB):
-            pass
\ No newline at end of file
+            pass
+
+- case: model_serializer_meta_attributes
+  main: |
+    from rest_framework import serializers
+
+    reveal_type(serializers.ModelSerializer.Meta.model) # N: Revealed type is 'Type[django.db.models.base.Model]'
+    reveal_type(serializers.ModelSerializer.Meta.fields) # N: Revealed type is 'Union[typing.Sequence[builtins.str], None]'
+    reveal_type(serializers.ModelSerializer.Meta.read_only_fields) # N: Revealed type is 'Union[typing.Sequence[builtins.str], None]'
+    reveal_type(serializers.ModelSerializer.Meta.exclude) # N: Revealed type is 'Union[typing.Sequence[builtins.str], None]'
+    reveal_type(serializers.ModelSerializer.Meta.depth) # N: Revealed type is 'Union[builtins.int, None]'
+    reveal_type(serializers.ModelSerializer.Meta.extra_kwargs) # N: Revealed type is 'builtins.dict[builtins.str, builtins.dict[builtins.str, Any]]'
+
+- case: test_model_serializer_passes_check
+  main: |
+    from rest_framework import serializers
+    from django.contrib.auth.models import User
+    from typing import Type
+
+    class TestSerializer(serializers.ModelSerializer):
+        class Meta:
+          model = User
+
+    def is_meta_model(serializer: Type[serializers.ModelSerializer]):
+        return bool(serializer.Meta.model)
+
+    is_meta_model(TestSerializer)
```

Comment:
Let's `reveal_type` here that it is `bool` and we are done!

Reply:
should be good now